src/main/java/org/moltimate/moltimatebackend/util/FileUtils.java

```diff
@@ -116,6 +113,22 @@ public static MotifFile readMotifFile(MultipartFile file) {
         }
     }
 
+    public static Structure getStructureFromFile(MultipartFile file) {
```

Comment:
Previously this method called readMotifFile ignoring the cases were an uploaded structure file can be a pdb or mmcif file
```
public static Structure getStructureFromFile(MultipartFile file) {
        return readMotifFile(file).getStructure();
}
```